exercise7.js

```diff
@@ -0,0 +1,8 @@
+
+// function findProduct(x,y){
+//     return x*y;
+
+//   }
+
+let product = (x,y) => x*y;
```

Comment:
It would be better if you made the product function definition const for better practice.